### DISCUSSION TITLE
Support for HomeAssistant OS on reComputer R1000

### DISCUSSION THREAD

**Discussion by DamirIbrisimovic** [Seeed-Studio/OSHW-RPi-Series#15] [Accomplished]
I want to start a discussion about integrating HomeAssistant on the SeeedStudio R1000. This device offers excellent communication capabilities with KNX and ModBus systems. If HomeAssistant could run on the R1000, it would simplify the connection and control of smart homes.

**Comment by J-Paul0815** [comment]
I don't know the LoraWan support for HomeAssistant, but I know the LoraWan support for IoBroker all the better. It would certainly be interesting to test the reComputer R1000 with IoBroker and the LoraWan adapter. Thanks to the pre-configured downlink configuration, not only do the incoming uplinks work for many devices, but they can also be controlled immediately via downlinks without any further steps, for example a new value for the uplink interval.

**Comment by LJ-Hao** [comment]
If it succeeds, it would be a good idea to add my AI kit to the reComputer R1000 to achieve a seamless computer vision experience.

**Comment by KasunThushara** [comment]
Hi friends! We've just created a new wiki to enable Modbus RS485 functionalities. Alongside this, we've developed a couple of wikis on Node-RED and N3uron. With this motivation, we're now working on integrations with Home Assistant and Modbus. Stay tuned for updates!!!

**Comment by KasunThushara** [comment]
Hi Friends, I have installed HA on reComputer R1000 in the supervised matter. cause I want to use reComputer for other operations. I was abled to integrate my PLC with Modbus and HA. 
![HA modbus](https://github.com/user-attachments/assets/42c64caf-4b8c-47e6-8ee5-c65a5f3f8167)

**Reply by KasunThushara** [reply]
The detailed WiKi can be found in here: https://wiki.seeedstudio.com/recomputer_r1000_home_assistant_modbus/
